integrations/google_genai/src/haystack_integrations/components/generators/google_genai/chat/chat_generator.py

```diff
@@ -44,7 +44,6 @@
     "PROHIBITED_CONTENT": "content_filter",
     "SPII": "content_filter",
     "IMAGE_SAFETY": "content_filter",
-    "FUNCTION_CALL": "tool_calls",
```

Comment:
I found that in #2078 I erroneously introduced this line, but Google GenAI does not have `genai.types.FinishReason. FUNCTION_CALL`. When the model produces a tool call, the finish reason is `genai.types.FinishReason. STOP`. https://googleapis.github.io/python-genai/genai.html#genai.types.FinishReason

I would prefer to stick to Google conventions and not implement logic to identify tool calls and remap the finish reason, but we can discuss this point.

Reply:
Huh that is a bit of a bummer about the finish reason being stop for tool calls. 

I think fine to leave as-is for now, but let's open an issue to discuss. I think ideally we would identify the finish reason as tool calls in that scenario but I could see that being some work.

Reply:
Ok. I will open an issue.

Reply:
https://github.com/deepset-ai/haystack-core-integrations/issues/2143